.github/workflows/mvn-verify-check.yml

```
@@ -42,10 +42,8 @@ jobs:
         id: noSnapshotVersionsStep
         run: |
           set -x
-          noSnapshotVersionsStr=$(mvn -B help:evaluate -q -pl dist -PnoSnapshots -Dexpression=included_buildvers -DforceStdout)
-          noSnapshotVersionsStr=$(echo $noSnapshotVersionsStr)
-          noSnapshotVersionsArr=($(IFS=", "; echo $noSnapshotVersionsStr))
-          tailNoSnapshotVersionsArr=(${noSnapshotVersionsArr[@]:1})
+          . $(dirname "$0")/../../jenkins/version-def.sh
```

given that maven commands work this stance is executed against the repo root directory. This should be 
```suggestion
          . ./jenkins/version-def.sh
```